src/controller/project_controller/project_controller.py

```diff
@@ -0,0 +1,47 @@
+from flask import request, jsonify
+from src.dao.project_dao.project_dao import ProjectDAO
+from src.dto.project_dto.project_dto import Project
+from src.http_status_code_enum import HttpCode
+class ProjectController:
+
+    projectDAO = ProjectDAO()
+
+    def __init__(self) -> None:
+        pass
+
+    def create_project(self,json):
+        project_name = json["project_name"]
+        status = json["status"]
+        content = json["content"]
+        # fake id to init project object
+        project = Project(1,project_name, status, content)
```

Comment:
you look at weiyang code. He got a better way to directly parse in the request.json to create the DTO object. His way is better.

Reply:
noted, thanks!